docs/api-reference/io-columns-clustering.md

```diff
@@ -3,5 +3,5 @@ The input features column data must be <xref:System.Single>. No label column nee
 
 | Output Column Name | Column Type | Description|
 | -- | -- | -- |
-| `Score` | <xref:System.Single> | The unbounded score that was calculated by the trainer to determine the prediction.|
-| `PredictedLabel` | <xref:System.Int32> | The cluster id predicted by the trainer.|
\ No newline at end of file
+| `Score` | <xref:System.Single> | The unbounded score that was calculated by the trainer to determine the prediction. |
```

Comment:
>to determine the prediction [](start = 90, length = 27)

I would change to "to determine which cluster the example belongs to".

Reply:
Ok.

---
In reply to: [277750939](https://github.com/dotnet/machinelearning/pull/3520#discussion_r277750939) [](ancestors = 277750939)